configuration/scripts/machines/Macros.freya_intel

```diff
@@ -0,0 +1,75 @@
+#==============================================================================
+# Makefile macros for DMI Freya based on ECCC banting
+#==============================================================================
+# For use with intel compiler
+#==============================================================================
+
+#INCLDIR := -I. -I/usr/include
+#SLIBS   :=
+
+#--- Compiler/preprocessor ---
+FC         := ftn 
+CC         := cc
+CXX        := CC
+CPP        := /usr/bin/cpp
+CPPFLAGS   := -P -traditional  # ALLOW fortran double backslash "\\"
+SCC   := gcc
+SFC   := ftn
+
+CPPDEFS    := -DFORTRANUNDERSCORE ${ICE_CPPDEFS}
+CFLAGS     := -c -O2 -fp-model precise
+
+# Additional flags
+FIXEDFLAGS := -132
+FREEFLAGS  := -FR
+FFLAGS     := -fp-model source -convert big_endian -assume byterecl -ftz -traceback -no-wrap-margin
+#-xHost
+
+ifeq ($(ICE_BLDDEBUG), true)
+  FFLAGS     += -O0 -g -check -fpe0 -ftrapuv -fp-model except -check noarg_temp_created
+# -heap-arrays 1024 
+else
+  FFLAGS     += -O2
+endif
+LD         := $(FC)
+LDFLAGS    := $(FFLAGS) -v
+#ifeq ($(ICE_BLDDEBUG), true)
+#FFLAGS     := -O0 -g -check uninit -check bounds -check pointers -fpe0 -check noarg_temp_created
+#FFLAGS     := -g -O0 -traceback -fp-model precise -fp-stack-check -fpe0
+#else
+#FFLAGS     := -r8 -i4 -O2 -align all -w -ftz -assume byterecl
+# FFLAGS     := -O2 -fp-model precise -assume byterecl -ftz -traceback -xHost
+#endif
+# Preprocessor flags
+#CPPDEFS    := -DLINUX $(ICE_CPPDEFS)
+
+# Linker flags
+
+# Additional flags
+
+ifeq ($(DITTO), yes)
+   CPPDEFS :=  $(CPPDEFS) -DREPRODUCIBLE
+endif
+
+
+ifeq ($(ICE_THREADED), true) 
+   LDFLAGS += -qopenmp 
+   CFLAGS += -qopenmp 
+   FFLAGS += -qopenmp 
+endif
+
+#--- NetCDF ---
+#ifeq ($(IO_TYPE), netcdf)
+#   
+#endif
+#
+#ifeq ($(IO_TYPE), netcdf_bin)
+#   CPPDEFS :=  $(CPPDEFS) -Dncdf
+#endif
+
+### if using parallel I/O, load all 3 libraries.  PIO must be first!
```

Comment:
this part should have been removed long ago from our Macro files, PIO is not installed on our machines... So I would suggest removing it also here, to not carry uneeded stuff.

Reply:
gcc: The short answer makdep fails if I use cc. It works when I use gcc. I think, but I am not sure, that the reason is that the XC50 computer at DMI consist of login nodes and compute nodes, which have different architectures. The cc command is used when compiling for compute nodes and gcc for login nodes.
I will separate the computers within the if statement and update.
I agree about the DITTO and REPRODUCIBLE but I left them in there to be sure.
There is an installation of parallel netcdf on the DMI computer but I am not sure if it works. I have never tried CICE or other model systems with pio.

Reply:
ah yes, makes sense to use `gcc` if the login and computes have different architectures. I think you also could use the `CFLAGS_HOST` macro for that : https://cice-consortium-cice.readthedocs.io/en/latest/user_guide/ug_running.html#cross-compiling

This would be a little "cleaner" I think, since this way the C code (ice_reprosum..) would still be compiled by `cc` (I don't know if it would really make a difference though.)